pyvista/plotting/mapper.py

```diff
@@ -303,19 +303,14 @@ def set_custom_opacity(
         ):
             """Set custom opacity."""
             # create a custom RGBA array to supply our opacity to
-            if opacity.size == mesh.n_points and opacity.size == mesh.n_cells:
-                if preference == 'points':
-                    rgba = np.empty((mesh.n_points, 4), np.uint8)
-                else:
-                    rgba = np.empty((mesh.n_cells, 4), np.uint8)
-            elif opacity.size == mesh.n_points:
+            if opacity.size == mesh.n_points:
                 rgba = np.empty((mesh.n_points, 4), np.uint8)
             elif opacity.size == mesh.n_cells:
                 rgba = np.empty((mesh.n_cells, 4), np.uint8)
             else:
                 raise ValueError(
                     f"Opacity array size ({opacity.size}) does not equal "
-                    f"the number of points {mesh.n_points} or the "
+                    f"the number of points ({mesh.n_points}) or the "
                     f"number of cells ({mesh.n_cells})."
                 )
```

Comment:
This case is not covered, but as far as I can tell this is never reached because there's an error raised earlier in `process_opacity()`. I didn't add a no-cover pragma here because I'm not 100% certain.

Reply:
I'm seeing a 100% patch coverage.

Reply:
Hmm, good point, curious. In a local coverage run I see red here. And it seems so does codecov:
![Screenshot from 2022-06-18 17-28-54](https://user-images.githubusercontent.com/17914410/174445434-9ee2d15f-b1ce-4aea-a327-401a81f69594.png)

I assume because the diff is on one line of a multiline statement, it doesn't consider it for diff coverage. The red at the `raise ValueError` line still suggests that this is not covered.

Reply:
Weird. `codecov` should have at least complained about this considering that the line has been modified but the tests don't cover this.

Ignoring coverage in 41f4d1b76356be691b018b0422fc8a7da11fcb7f.